packages/babel-helpers/test/helpers/define-helper.js

```diff
@@ -0,0 +1,15 @@
+import template from "babel-template";
+import helpers from "../../lib/helpers";
+
+// This is not garanteed to be 100% unique, but I hope
+// no one will ever name a babel helper like "_$_$_$_foo_12"!
+let i = 0;
```

Comment:
Does having this increment like this mean that if we run the tests in a different order, we'll end up with different IDs?

I also have mixed feelings about these not deleting the helpers they add after the test has completed.

Reply:
> Does having this increment like this mean that if we run the tests in a different order, we'll end up with different IDs?

Yes, but it isn't observable in `expected.js` because the only part of the ID which changes is a number and thus it is stripped away by `.generateUidIdentifier`. Anyway, I will make the helper name deterministic by using the test name instead of the `i` variable.

> I also have mixed feelings about these not deleting the helpers they add after the test has completed.

I could delete them after each fixture, but I would need to implement a custom fixtures runner or mock `babel.transform` to remove them after the transformation is finished. I can't do it in a `Program#exit` visitor because it isn't called if the test throws.

I don't think not sharing state between tests is worth this additional complexity, since the test helpers' names are unique and thus they can't conflict.

If you think there is a simpler way to remove the registered helpers, please let me know.